src/applications/facility-locator/components/ResultsList.jsx

```diff
@@ -19,7 +26,8 @@ class ResultsList extends Component {
   shouldComponentUpdate(nextProps) {
     return (
       nextProps.results !== this.props.results ||
-      nextProps.inProgress !== this.props.inProgress
+      nextProps.inProgress !== this.props.inProgress ||
+      nextProps.searchString !== this.props.searchString
```

Comment:
Without this addition, the loading indicator on page load would be missing the search query. This was really noticeable search Community Cares. It would say `Searching for Community Care (Non-VA Health) in` and that's all.